infra/build/build_status/fuzz_introspector_page_gen.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/python3
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Logic to create Fuzz Introspector overview page."""
+import json
+
+from urllib.request import urlopen
+from bs4 import BeautifulSoup
+
+TABLE_HEAD = """<table id="mainTable">
+<thead>
+            <tr>
+      <th title="Fuzzer key. Usually fuzzer executable file">
+       Project Report
+      </th>
+      <th title="Number of fuzzers for the project.">
+       Fuzzer count
```

Comment:
Nit: can we rename this to "fuzz target" to be consistent with our other documentation?

Reply:
I edited this myself.